[ bug ] Docker OpenJDK Image Not Found

## 🛠️ 발견된 버그 기능
[//]: # (어떤 부분에서 버그가 나오는지 적습니다.)
기존의 워크플로우 파일에서 Dockerfile 부분이 에러가 발생함 (아래는 에러 메시지)


## 🌎 발견된 환경
[//]: # (어떤 환경에서 버그가 나오는지 적습니다.)
Docker Hub에서 openjdk 공식 저장소가 더 이상 지원되지 않기 때문에(deprecated) 발생함. openjdk:17-jdk-slim 같은 태그들이 삭제되었거나 업데이트되지 않아 이미지를 찾을 수 없음.
```
Run echo "***" | docker login -u "***" --password-stdin
Login Succeeded
WARNING! Your credentials are stored unencrypted in '/home/runner/.docker/config.json'.
Configure a credential helper to remove this warning. See
https://docs.docker.com/go/credential-store/
#0 building with "default" instance using docker driver
#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 184B done
#1 DONE 0.0s
#2 [auth] library/openjdk:pull token for registry-1.docker.io
#2 DONE 0.0s
#3 [internal] load metadata for docker.io/library/openjdk:17-jdk-slim
#3 ERROR: docker.io/library/openjdk:17-jdk-slim: not found
***
 > [internal] load metadata for docker.io/library/openjdk:17-jdk-slim:
***
Dockerfile:1
***
   1 | >>> FROM openjdk:17-jdk-slim
   2 |     
   3 |     WORKDIR /app
***
ERROR: failed to build: failed to solve: openjdk:17-jdk-slim: failed to resolve source metadata for docker.io/library/openjdk:17-jdk-slim: docker.io/library/openjdk:17-jdk-slim: not found
Error: Process completed with exit code 1.
```


## 📝 check-lists
- [ ] 대체 이미지 확인

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,9 +1,11 @@
-FROM openjdk:17-jdk-slim
+FROM eclipse-temurin:17-jre-jammy
 
 WORKDIR /app
 
 COPY build/libs/admin_be-0.0.1-SNAPSHOT.jar app.jar
 
 EXPOSE 8080
+
+ENV TZ=Asia/Seoul
 
 ENTRYPOINT ["java", "-jar", "app.jar"]

